Fancy readme (#25)
sornas/lingon: README.md

# Lingon - The Sour Game Engine

> Lingon is a minimalistic game engine aimed at Game Jams, made to be compatibly
> with the sylt programming language.

Reviewer: ```suggestion
Lingon is a minimalistic game engine aimed at Game Jams, made to be compatible
with the [sylt programming language](https://github.com/fredthedino/sylt-lang/).
```

*NOTE: Under heavy development*

## Getting started
There are two ways this engine can be used:
1. Depend on this repository in your Cargo.toml and setup a minimal project (see main.rs) 

> 2. Clone thie repository and write the game in the engine

Reviewer: ```suggestion
2. Clone this repository and write the game in the engine
```

## Some nice features
 - Sylt integration, for fast prototyping

>  - Particles, lots and lots of particles

Reviewer: ```suggestion
 - Particles. Lots and lots of particles
```

>  - Fast build times, dependencies are kept to a minimum to make it quick to get started

Reviewer: ```suggestion
 - Fast build times. Dependencies are kept to a minimum to make it quick to get started
```

> <img src="lingon.svg" width="4em"/>

Author: ```suggestion
<img src="lingon.svg" width="256"/>
```

> <img src="lingon.svg" width="256"/>

Author: ```suggestion
<img src="lingon.svg" width="64"/>
```
